.github/workflows/wheels.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           . venv/bin/activate
           python --version
           pip install "$(grep '^uv' < requirements.txt)"
-          uv pip install -r requirements.txt
+          uv pip install -r requirements.txt ninja
```

Comment:
Was failing on armhf.... not sure this makes sense.. test this and will remove it